src/09-word-problems.js

```diff
@@ -19,7 +19,58 @@
  *  applyDiscount(1000, 9, true);
  *  //> 700
  */
-function applyDiscount(priceInCents, age, hasMembership) {}
+//10 yrs old or younger OR 65 yrs old or older = 10% discount; attending w/ memember of memebership =20% off; both apply = 30% off
+function applyDiscount(priceInCents, age, hasMembership) {
+
+  if (hasMembership = priceInCents * 0.10 || priceInCents * 0.20 || priceInCents * 0.30) {
+    return priceInCents//this line passed in test run only
+  } else if (age <= 10 || age >= 65 && priceInCents * 0.10) {
+    return priceInCents
+  } else if (hasMembership === true && priceInCents * 0.20) {
+    return priceInCents
+  } else if (hasMembership && age <= 10 || age >= 65 || priceInCents * 0.30) {
+    return priceInCents
+  } else {
+    return priceInCents
+  }
+}
```

Comment:
If you run the debugger on this problem using the input from the first failing test, you'll find some surprising things about which `if` conditions are evaluating to `true` and how the value of `priceInCents` changes before you return it.